app/main.py

```diff
@@ -1 +1,26 @@
-# write your code here
+import shutil
+
+
+def copy_file(command: str) -> None:
+    parts = command.split()
+
+    if len(parts) == 3 and parts[0] == "cp":
```

Comment:
`if len(parts) == 3 and parts[0] == "cp" and old_file != new_file:`
...rest of logic here